src/systems/cameras.js

```diff
@@ -0,0 +1,28 @@
+// Used for tracking and managing camera tools in the scene
+AFRAME.registerSystem("cameras", {
```

Comment:
ooh, while reading all the rest of the code I was assuming this was the system for the build in aframe `camera` component (which is also called `camera` so I am surprised this didn't break everything). We should call this `camera-tool`, it also then gets automatically bound in `camera-tool` component as `this.system`